swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -466,13 +464,11 @@ final class IterableAPIInternal: NSObject, PushTrackerProtocol, AuthProvider {
     private func storeAuthData() {
```

Comment:
Since we are not storing authToken here anymore maybe we should rename this method accordingly? `storeUserCredentials`? (Not sure about this name either).

Reply:
I chose `storeIdentifierData`, I think it's more fitting?